### ISSUE TITLE
Marathon engineering is horrible

### ISSUE THREAD

**Issue by Just-a-Unity-Dev** [space-wizards/space-station-14#29776]
## Description
I'm going to refer to BOTH the singulo/tesla as the singulo because I'm on a phone really mad at marathon's design

TL;DR: Marathon has constant power issues because (a.) it doesn't have it's singulo encasing pre-built and (b.) doesn't have enough power round start to run the station before getting singulo set up which causes head splitting headaches to any poor engineer(s) that even dare try and play on this map.

For a map that is supposed to be powered by the singulo it is a pain to do so. Building the singulo with parts can take a good 15-30 minutes and *sourcing* the parts is an issue in on itself because unless cargo has the mats for it the materials that come scattered across engineering are usually never enough, not to mention engineering has to literally build the entire singulo by themselves with no outer protection towards asteroids. I don't even think you can set up the singulo without buying AME refuels from cargo. When I last played on Marathon it took a good hour for engineering to get everything done (by the way: marathon only supports _only four engineers_ minus CE!) and by the time that singulo was set up, evac had already arrived.

BLUF

**Comment by Just-a-Unity-Dev** [comment]
sidenote: this doesn't include atmos because I don't play atmos and thus I don't have any thoughts on atmos

**Comment by deltanedas** [comment]
just set up teg

**Comment by Emisse** [comment]
needs standardization with other engineering depts, i will get to this sometime unless someone else wants to do it sooner

**Comment by osjarw** [comment]
Tried to build it myself just now as a solo engi, it took me 29 minutes. I had the AME power the station for that duration, with it being slightly overclocked. 

It looks like there is _mostly_ enough materials to build the setup, though sourcing the materials can indeed be hard. 
Engi seems to have 3 stacks of steel, of which you need ~1.5 to build the PA itself if I counted correctly. There are 3 more stacks in atmos. So you might have troubles with sourcing steel if rest of engineering isn't cooperative. I guess you could get some steel from deconstructing the crates at the construction area.

The machines needed to construct the setup can be found in the construction area and the vault/room behind blast doors in engi. 

The only thing you may need the help of cargo for is to build the PA fuel chamber, which reguires an AME fuel jar, which might be hard to source, since this station seems to start with just a single one. However, it looks like you can use spent fuel jars, though that may be a bit stressful, since afaik, the default SMES bank lasts for like 3 minutes. 

Also cargo might be needed in order to get more tesla coils, since you start with just a single one. Also I don't really know if 3 radiation collectors is enough to power the station. 

I think most of the issues can be solved by giving engi some more steel, additional AME fuel jar, and a few more tesla coils, since a single one seems to be having lots of trouble in powering the station. Also additional grounding rods could be useful, since you start with just a single one.

**Comment by thebadman4662** [comment]
> just set up teg(and maybe ask cargo for 3ish+ rad collectors)

Just set up damaged RTG. :godmode: 
*hides before wrath of PJB*

**Comment by deltanedas** [comment]
damaged rtg and 3 collectors making a mind boggling 55kw peak power :trollface:

**Comment by thebadman4662** [comment]
Well, you could ask boys to bring canisters to cargo with you and you will have 1 new collector per one in no time, if cargo is not being 100% antags as usual.

**Comment by Just-a-Unity-Dev** [comment]
> just set up teg

There was a discussion/message in #cartography on Discord about how the TEG was impossible to set up as a solo atmos before the AME burns out due to lack of fuel or exploding due to overclocking. Not everyone has hyperoptimized TEG setups burned inside of their brain.

**Comment by deltanedas** [comment]
then give it an actually good roundstart setup to skip 10 minutes of welding
